doc/source/whatsnew/v0.24.0.txt

```diff
@@ -80,6 +80,8 @@ Documentation Changes
 Bug Fixes
 ~~~~~~~~~
 
+- The top level None value is not dropped but rather preserved along with lower levels for consistency (:issue:`21356`)
```

Comment:
Added the bugfix in the whats new note.

Reply:
Change of plans here - can you add to the "Fixed Regressions" section in v0.23.1 introduced via #21368?

Reply:
this also needs to mention json_normalize, add double backticks on ``None``. 

finally, as a user this is not really clear on what has changed, can you re-word a bit.